server/sonar-web/src/main/js/apps/projects/filters/Filter.js

```diff
@@ -96,7 +115,7 @@ export default class Filter extends React.Component {
   renderOptions () {
     return (
         <div className="search-navigator-facet-list">
-          {this.props.options.map(option => this.renderOption(option))}
+          {this.props.getOptions(this.props.facet).map(option => this.renderOption(option))}
```

Comment:
@gregaubert because now the list of options is dynamic, it can be empty. In this case I'd display a message like "no results found".